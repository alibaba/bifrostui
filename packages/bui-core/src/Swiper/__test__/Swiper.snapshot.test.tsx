import { snapshotTest } from 'testing';

describe('Swiper snapshot', () => {
  snapshotTest('Swiper');
});
