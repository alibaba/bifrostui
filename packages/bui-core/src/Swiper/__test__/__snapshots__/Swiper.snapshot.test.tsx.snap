// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Swiper snapshot Swiper demo snapshot 0 1`] = `
<div
  className="swiper bui-swiper"
  style={
    Object {
      "--indicator-color": "rgba(255, 255, 255, .4)",
      "--indicator-color-active": "var(--bui-color-bg-view)",
      "height": "200px",
      "overflow": "hidden",
      "paddingBottom": 0,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": 0,
      "width": "300px",
    }
  }
>
  <div
    className="swiper-wrapper"
  >
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "lightgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        1
      </div>
    </div>
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "gray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        2
      </div>
    </div>
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "darkgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        3
      </div>
    </div>
  </div>
</div>
`;

exports[`Swiper snapshot Swiper demo snapshot 0 2`] = `
Array [
  <div
    className="bui-stack"
    style={
      Object {
        "--align-items": "center",
        "--flex-direction": "row",
        "--flex-wrap": "nowrap",
        "--justify-content": "center",
        "--spacing": "8px",
      }
    }
  >
    <button
      className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
      onClick={[Function]}
      type="button"
    >
      切换到1
    </button>
    <button
      className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
      onClick={[Function]}
      type="button"
    >
      切换到2
    </button>
    <button
      className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
      onClick={[Function]}
      type="button"
    >
      切换到3
    </button>
  </div>,
  <div
    className="swiper bui-swiper"
    style={
      Object {
        "--indicator-color": "rgba(255, 255, 255, .4)",
        "--indicator-color-active": "var(--bui-color-bg-view)",
        "height": "200px",
        "overflow": "hidden",
        "paddingBottom": 0,
        "paddingLeft": undefined,
        "paddingRight": undefined,
        "paddingTop": 0,
        "width": "300px",
      }
    }
  >
    <div
      className="swiper-wrapper"
    >
      <div
        className="swiper-slide"
      >
        <div
          className="bui-stack"
          style={
            Object {
              "--align-items": "center",
              "--flex-direction": "column",
              "--flex-wrap": "nowrap",
              "--justify-content": "center",
              "--spacing": "0",
              "background": "lightgray",
              "height": "100%",
              "width": "100%",
            }
          }
        >
          1
        </div>
      </div>
      <div
        className="swiper-slide"
      >
        <div
          className="bui-stack"
          style={
            Object {
              "--align-items": "center",
              "--flex-direction": "column",
              "--flex-wrap": "nowrap",
              "--justify-content": "center",
              "--spacing": "0",
              "background": "gray",
              "height": "100%",
              "width": "100%",
            }
          }
        >
          2
        </div>
      </div>
      <div
        className="swiper-slide"
      >
        <div
          className="bui-stack"
          style={
            Object {
              "--align-items": "center",
              "--flex-direction": "column",
              "--flex-wrap": "nowrap",
              "--justify-content": "center",
              "--spacing": "0",
              "background": "darkgray",
              "height": "100%",
              "width": "100%",
            }
          }
        >
          3
        </div>
      </div>
    </div>
  </div>,
]
`;

exports[`Swiper snapshot Swiper demo snapshot 0 3`] = `
<div
  className="swiper bui-swiper"
  style={
    Object {
      "--indicator-color": "rgba(255, 255, 255, .4)",
      "--indicator-color-active": "var(--bui-color-bg-view)",
      "height": "200px",
      "overflow": "hidden",
      "paddingBottom": 0,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": 0,
      "width": "300px",
    }
  }
>
  <div
    className="swiper-wrapper"
  >
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "lightgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        1
      </div>
    </div>
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "gray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        2
      </div>
    </div>
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "darkgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        3
      </div>
    </div>
  </div>
</div>
`;

exports[`Swiper snapshot Swiper demo snapshot 0 4`] = `
<div
  className="swiper bui-swiper"
  style={
    Object {
      "--indicator-color": "rgba(255, 255, 255, .4)",
      "--indicator-color-active": "var(--bui-color-bg-view)",
      "height": "200px",
      "overflow": "hidden",
      "paddingBottom": 0,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": 0,
      "width": "300px",
    }
  }
>
  <div
    className="swiper-wrapper"
  >
    <div
      className="swiper-slide swiper-slide-duplicate"
      data-swiper-slide-index={2}
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "darkgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        3
      </div>
    </div>
    <div
      className="swiper-slide"
      data-swiper-slide-index={0}
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "lightgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        1
      </div>
    </div>
    <div
      className="swiper-slide"
      data-swiper-slide-index={1}
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "gray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        2
      </div>
    </div>
    <div
      className="swiper-slide"
      data-swiper-slide-index={2}
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "darkgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        3
      </div>
    </div>
    <div
      className="swiper-slide swiper-slide-duplicate"
      data-swiper-slide-index={0}
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "lightgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        1
      </div>
    </div>
  </div>
</div>
`;

exports[`Swiper snapshot Swiper demo snapshot 0 5`] = `
<div
  className="swiper bui-swiper"
  style={
    Object {
      "--indicator-color": "rgba(255, 255, 255, .4)",
      "--indicator-color-active": "var(--bui-color-bg-view)",
      "height": "200px",
      "overflow": "hidden",
      "paddingBottom": 0,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": 0,
      "width": "300px",
    }
  }
>
  <div
    className="swiper-wrapper"
  >
    <div
      className="swiper-slide swiper-slide-duplicate"
      data-swiper-slide-index={2}
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "darkgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        3
      </div>
    </div>
    <div
      className="swiper-slide"
      data-swiper-slide-index={0}
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "lightgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        1
      </div>
    </div>
    <div
      className="swiper-slide"
      data-swiper-slide-index={1}
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "gray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        2
      </div>
    </div>
    <div
      className="swiper-slide"
      data-swiper-slide-index={2}
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "darkgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        3
      </div>
    </div>
    <div
      className="swiper-slide swiper-slide-duplicate"
      data-swiper-slide-index={0}
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "lightgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        1
      </div>
    </div>
  </div>
</div>
`;

exports[`Swiper snapshot Swiper demo snapshot 0 6`] = `
<div
  className="swiper bui-swiper"
  style={
    Object {
      "--indicator-color": "rgba(255, 255, 255, .4)",
      "--indicator-color-active": "var(--bui-color-bg-view)",
      "height": "200px",
      "overflow": "hidden",
      "paddingBottom": undefined,
      "paddingLeft": 0,
      "paddingRight": 0,
      "paddingTop": undefined,
      "width": "300px",
    }
  }
>
  <div
    className="swiper-wrapper"
  >
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "lightgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        1
      </div>
    </div>
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "gray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        2
      </div>
    </div>
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "darkgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        3
      </div>
    </div>
  </div>
</div>
`;

exports[`Swiper snapshot Swiper demo snapshot 0 7`] = `
<div
  className="swiper bui-swiper"
  style={
    Object {
      "--indicator-color": "#880000",
      "--indicator-color-active": "#FF0000",
      "height": "200px",
      "overflow": "hidden",
      "paddingBottom": 0,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": 0,
      "width": "300px",
    }
  }
>
  <div
    className="swiper-wrapper"
  >
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "lightgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        1
      </div>
    </div>
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "gray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        2
      </div>
    </div>
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "darkgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        3
      </div>
    </div>
  </div>
  <div
    className="swiper-pagination"
  />
</div>
`;

exports[`Swiper snapshot Swiper demo snapshot 0 8`] = `
<div
  className="swiper bui-swiper"
  style={
    Object {
      "--indicator-color": "rgba(255, 255, 255, .4)",
      "--indicator-color-active": "var(--bui-color-bg-view)",
      "height": "200px",
      "overflow": "hidden",
      "paddingBottom": 0,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": 0,
      "width": "300px",
    }
  }
>
  <div
    className="swiper-wrapper"
  >
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "lightgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        1
      </div>
    </div>
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "gray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        2
      </div>
    </div>
    <div
      className="swiper-slide"
    >
      <div
        className="bui-stack"
        style={
          Object {
            "--align-items": "center",
            "--flex-direction": "column",
            "--flex-wrap": "nowrap",
            "--justify-content": "center",
            "--spacing": "0",
            "background": "darkgray",
            "height": "100%",
            "width": "100%",
          }
        }
      >
        3
      </div>
    </div>
  </div>
</div>
`;
